### PR TITLE
now support YYYY/MM/dd in track command

### PR DESCRIPTION
**Pull request by StephenMPapadatos**
Now can submit a track command that looks like this:
@togglbot track breatheagain 9:00AM-5:00PM 2017/08/05 BAS-79
or
@togglbot track breatheagain 9:00AM-5:00PM 2017/8/5 BAS-79
or if for today
@togglbot track breatheagain 9:00AM-5:00PM today BAS-79

either date (YYYY/MM/dd) or "today" must be inserted for track command

### CODE REVIEW

**Comment by StephenMPapadatos** [comment]
relates to issue #7

File: temp.go [hunk 1]
```diff
@@ -213,19 +214,22 @@ func handleBotCommands(replyChannel chan ReplyChannel) {
...
-				reply.DisplayTitle = "Sorry, I don't have enough information to make an event for you. try `@togglbot track PROJECT_NAME 9:00AM-5:00PM TASK_DESCRIPTION`"
+				reply.DisplayTitle = "Sorry, I don't have enough information to make an event for you. try `@togglbot track PROJECT_NAME 9:00AM-5:00PM YYYY/MM/DD (or 'today') TASK_DESCRIPTION`"
 				replyChannel <- reply
 				break
 			}
 			projectName := commandArray[1]
 			pid := getProjectWithName(togglApiKey, projectName)
 			timeRange := commandArray[2]
+
+			parsedDate, err := parseDate(commandArray[3])
```
Comment (tadgh) [inline temp.go:225]: this can throw an index error in case of this legal command: 
`track breatheagain 9:00AM-5:00PM`

**Comment by tadgh** [comment]
Looks good, but can you possibly make it so that if the date is omitted, today is assumed? Instead of having to input "today"? as it stands, this pr would break the existing track workflow. 

e.g, if today is august 12th, 2017

`track PROJECT_NAME 9:00AM-5:00PM today`

should evaluate the same as 

`track PROJECT_NAME 9:00AM-5:00PM`

since both date and description should be optional.

File: temp.go (hunk 1 above)
Comment (tadgh) [reply temp.go:225]: Ah actually, i suppose thats not the case since description is optional, disregard.

**Comment by tadgh** [comment]
Also, please update the `help` command to reflect the changes.

**Pull request merged**